### PR TITLE
Try out AssetHub transfers with chopsticks

### PR DESCRIPTION
**Pull request by JuaniRios**
## What?
- Add chopstick config files to test asset hub transfers to Polimec

## Why?
- To make sure our live Polimec xcm config is correct

## How?
- add balances to the sovereign account and sender in assethub, and create the assets for DOT, USDC, and USDT on Polimec

## Testing?
USDT transfer encoded call to execute on AssetHub:
```
0x1f020101010041340100010100ba143e2096e073cb9cddc78e6f4969d8a02160d716a69e08214caf5339d88c420104000002043205011f000284d71700000000
```

### CODE REVIEW

**Comment by JuaniRios** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/Polimec/polimec-node/282?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#282** <a href="https://app.graphite.dev/github/pr/Polimec/polimec-node/282?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#281** <a href="https://app.graphite.dev/github/pr/Polimec/polimec-node/281?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @JuaniRios and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: scripts/chopsticks/assethub-transfers/polkadot-assethub.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+db: ./db.sqlite
+
```
Comment (lrazovic) [inline scripts/chopsticks/assethub-transfers/polkadot-assethub.yml:2]:
```suggestion
mock-signature-host: true
```

File: scripts/chopsticks/assethub-transfers/polkadot-assethub.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+          [1984, 15CyvMWHKwbAPQ3ekqRFWLyq3kcjU8ypMuuGfsMnxanW85WZ],
+          { balance: 1000000000 },
+        ],
+        [
+          [1337, 15CyvMWHKwbAPQ3ekqRFWLyq3kcjU8ypMuuGfsMnxanW85WZ],
+          { balance: 1000000000 },
+        ],
+      ]
+
+mock-signature-host: true
```
Comment (lrazovic) [inline scripts/chopsticks/assethub-transfers/polkadot-assethub.yml:30]:
```suggestion
```

File: scripts/chopsticks/assethub-transfers/polkadot-polimec.yml [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+          [1984, 15CyvMWHKwbAPQ3ekqRFWLyq3kcjU8ypMuuGfsMnxanW85WZ],
+          { balance: 1000000000 },
+        ],
+        [
+          [10, 15CyvMWHKwbAPQ3ekqRFWLyq3kcjU8ypMuuGfsMnxanW85WZ],
+          { balance: 1000000000 },
+        ],
+      ]
+
+mock-signature-host: true
```
Comment (lrazovic) [inline scripts/chopsticks/assethub-transfers/polkadot-polimec.yml:92]:
```suggestion
```

File: scripts/chopsticks/assethub-transfers/polkadot-polimec.yml [hunk 4]
```diff
@@ -0,0 +1,92 @@
+db: ./db.sqlite
+
```
Comment (lrazovic) [inline scripts/chopsticks/assethub-transfers/polkadot-polimec.yml:2]:
```suggestion
mock-signature-host: true
```

File: scripts/chopsticks/assethub-transfers/polkadot-polimec.yml [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+        [
+          [
+            "0xba143e2096e073cb9cddc78e6f4969d8a02160d716a69e08214caf5339d88c42",
+          ],
+          { providers: 1, data: { free: "10000000000000000" } },
+        ],
+      ]
+
+  ForeignAssets:
+    Asset:
```
Comment (lrazovic) [inline scripts/chopsticks/assethub-transfers/polkadot-polimec.yml:18]: Can you please add a comment remarking that this part of the `import-storage` config will be deleted as soon we configure the pallet on-chain?

**Review by lrazovic — APPROVED** [review]
Mega! I just have a few inline comments. Also, could you please create a simple `README.md` in the `scripts/chopsticks/assethub-transfers` directory to explain how these files should be used? Thanks!

**Pull request merged**